### DISCUSSION TITLE
dyndns

### DISCUSSION THREAD

**Discussion by dozwin81** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#446] [Q&A]
Hi, I have a dyndns and want to access the web interface on the go, which port do I have to forward in the router to get connected?

**Comment by thedavestewart** [comment]
EDIT: Do not use this solution. It won't work. 

Standard port 80, with some other external forwarding port (how to do it will be dependent on your router) ie:
External -- http://YOURID.dyndns.org:12345 --> Internal http://(yourlayzspaip):80

**Comment by dozwin81** [comment]
Why don't I see any data here? Locally I see everything?
![Frage1](https://user-images.githubusercontent.com/119894910/218049393-f03c061f-ad33-41c1-b330-b842e91528ef.jpg)

**Comment by dozwin81** [comment]
![Frage2](https://user-images.githubusercontent.com/119894910/218049869-8f2a468b-e050-4e74-bc53-fa16c86e8d30.jpg)

**Comment by thedavestewart** [comment]
So looking closer - there's also some websocket port requests that you'd need as well, but they're common DNS ports (and I wouldn't recommend opening those up directly, and the webui isn't going to know to forward them). Looking at some other posts, seems like the preferred method is to use a VPN to your home network and control as if it were a local device.

**Comment by jamesonuk** [comment]
opening ports to the internet if very dangerous and opening port 80 is a asking for trouble.
This would allow anyone to access the ESP (there is no access control) and there is a file upload page allowing them to essentially run anything they like on your network.   I have port 443 (https) open and I can see hundreds of attempts to scan the port, connect, look for vulnerabilities each day.

As @thedavestewart said a VPN is probably the best solution as this gives you encrypted / secured access to your network from outside.   They however generally a bit of a pain to setup.  A lot will depend on your router or whether you have anything else (an always on computer, raspberry pi etc.) you could use as your VPN endpoint.

**Reply by visualapproach** [reply]
There is a simple password check you can turn on in the config file but I wouldn't trust it to the internet
